### PR TITLE
chore: improvements for the marketplace presentation.

### PR DESCRIPTION
**Pull request by i4ki**

### CODE REVIEW

**Review by katcipis — APPROVED** [review]
LGTM + possibly dumb/useless comment

File: package.json [hunk 1]
```diff
@@ -1,8 +1,10 @@
 {
-	"name": "vscode-terramate",
+	"name": "terramate",
+  	"displayName": "Mineiros Terramate",
 	"publisher": "mineiros",
 	"description": "VSCode Terramate Extension",
 	"author": "Mineiros GmbH",
+	"icon": "assets/terramate.png",
```
[the remark this answers is not in the document]
Comment (i4ki) [reply package.json:7]: SVGs are forbidden in the Microsoft marketplace for security reasons...

```
Due to security concerns, vsce will not publish extensions that contain user-provided SVG images.
```
here: https://code.visualstudio.com/api/working-with-extensions/publishing-extension

[the remark this answers is not in the document]
Comment (katcipis) [reply package.json:7]: hmm was not aware of the dangers of SVGs :laughing:

File: package.json [hunk 2]
```diff
@@ -108,4 +110,4 @@
 		"mocha": "^9.2.1",
 		"typescript": "^4.6.2"
 	}
-}
+}
```
Comment (katcipis) [inline package.json:113]:
```suggestion
}
```

**Pull request merged**